src/client/home/components/LandingGraphicSilver.tsx

```diff
@@ -157,9 +157,8 @@ const LandingGraphicSliver = () => {
               color="primary"
               size="large"
               onClick={() =>
-                document
-                  .getElementById('landing-bottom')
-                  .scrollIntoView({ behavior: 'smooth' })
+                // ! show to TS that document will never be null
+                document.getElementById('landing-bottom')!.scrollIntoView({ behavior: 'smooth' })
```

Comment:
this invariant is only true right now with our current code. X months later if someone deletes the element, this assertion is going to come and bite us. Is it possible to use optional chaining here? `document.getelementbyid()?.scrollIntoView()` etc